github/ci/kubernetes_crud/vars/main.yml

```diff
@@ -0,0 +1,4 @@
+default_fatal_validation: yes
```

Comment:
is this intentional?

Reply:
The file was lost during the changes required for the merge of this PR #608, I recovered it verbatim. It means that by default a failure validating k8s resources before applying them would fail the operation.